Os/Pthreads/docs/sdd.md

```diff
@@ -213,4 +213,4 @@ Test complete.
 ---------------------
 ```
 
-The Pthreads queues are significantly faster than posix queues on a single core. The Pthreads queue is marginally slower in a multi-threaded environment, but its performance is still very similar to the Posix queue implementation. Based on these results the the Pthreads queue is performant enough for single core flight systems. It also looks to have reasonable performance when multi-threaded. If we wanted better multi-core performance, we could look into implementing a [lock-free concurrent max heap data structure](http://www.non-blocking.com/download/SunT03_PQueue_TR.pdf).
+The Pthreads queues are significantly faster than posix queues on a single core. The Pthreads queue is marginally slower in a multi-threaded environment, but its performance is still very similar to the Posix queue implementation. Based on these results the Pthreads queue is performant enough for single core flight systems. It also looks to have reasonable performance when multi-threaded. If we wanted better multi-core performance, we could look into implementing a [lock-free concurrent max heap data structure](http://www.non-blocking.com/download/SunT03_PQueue_TR.pdf).
```

Comment:
This paragraph is confusing. For example, the first sentence says that Pthreads queues are faster on a single core. But the second sentence says that Pthreads are slower in the presence of multiple threads. So what happens with multiple threads on one core? I suspect that all the instances of "multithreaded" in this paragraph should be replaced with "multicore" but I'm not sure.

Reply:
This change was just to remove the redundant "the", e.g. "the the" -> "the".  I don't have the context to rewrite this myself.